src/ol/source/ImageStatic.js

```diff
@@ -52,11 +50,23 @@ class Static extends ImageSource {
       projection: getProjection(options.projection)
     });
 
+    /**
+     * @private
+     * @type {string}
+     */
+    this.url_ = options.url;
+
+    /**
+     * @private
+     * @type {module:ol/extent~Extent}
+     */
+    this.imageExtent_ = options.imageExtent;
+
     /**
      * @private
      * @type {module:ol/Image}
      */
-    this.image_ = new ImageWrapper(imageExtent, undefined, 1, options.url, crossOrigin, imageLoadFunction);
+    this.image_ = new ImageWrapper(this.imageExtent_, undefined, 1, options.url, crossOrigin, imageLoadFunction);
```

Comment:
minor: use `this.url_` instead of `options.url` here

Reply:
Thanks for your feedback